lab_2/main.py

```diff
@@ -92,7 +216,36 @@ def find_diff_in_sentence(original_sentence_tokens: tuple, suspicious_sentence_t
     :param lcs: a longest common subsequence
     :return: a tuple with tuples of indexes
     """
-    pass
+    conditions = [checker(original_sentence_tokens, tuple),
+                  checker(suspicious_sentence_tokens, tuple),
+                  checker(lcs, tuple)]
+    if not(all(conditions)):
+        return ()
+
+    def count_difference(tokens):
```

Comment:
It is better not to nest functions. I suggest moving it outside and using it like utility function here.